apps/rewards-dashboard/src/components/Header.tsx

```diff
@@ -0,0 +1,87 @@
+//import { useWeb3Modal } from "@web3modal/wagmi/react"
+
+import { CiMenuBurger } from "react-icons/ci";
+import { WalletManager } from "../quarks/WalletManager";
+import { useState } from "react";
+import { MobileNavItem, NavItem } from "../quarks/NavItem";
+import { PortalLogoWithText } from "../quarks/LogoVectors";
+
+export const Header = () => {
+  const [showMenu, setShowMenu] = useState(false);
+  return (
+    <>
+      <div className="flex flex-col">
+        <div
+          className="
+        flex flex-row
+        gap-1
+        justify-between
+        items-center
+        "
+        >
+          <div
+            className="
+          flex flex-row gap-2
+          items-center
+          "
+          >
+            <PortalLogoWithText />
```

Comment:
could portal logo add a link to main app?

Reply:
good idea, done